bin/gwu/gwuLib.ml

```diff
@@ -948,34 +950,34 @@ let print_notes_for_person opts base gen p =
     Printf.ksprintf (oc opts) "beg\n";
     if notes <> "" then Printf.ksprintf (oc opts) "%s\n" notes;
     (if put_events_in_notes base p then
-     let rec loop pevents =
-       match pevents with
-       | [] -> ()
-       | evt :: events -> (
-           match evt.epers_name with
-           | Epers_Birth | Epers_Baptism | Epers_Death | Epers_Burial
-           | Epers_Cremation ->
-               let name =
-                 match evt.epers_name with
-                 | Epers_Birth -> "birth"
-                 | Epers_Baptism -> "baptism"
-                 | Epers_Death -> "death"
-                 | Epers_Burial -> "burial"
-                 | Epers_Cremation -> "creamation"
-                 | _ -> ""
-               in
-               let notes =
-                 if opts.no_notes <> `nnn then sou base evt.epers_note else ""
-               in
-               if notes <> "" then
-                 Printf.ksprintf (oc opts) "%s: %s\n" name notes;
-               print_witness_in_notes evt.epers_witnesses;
-               loop events
-           | _ ->
-               print_pevent opts base gen evt;
-               loop events)
-     in
-     loop (get_pevents p));
+       let rec loop pevents =
+         match pevents with
+         | [] -> ()
+         | evt :: events -> (
+             match evt.epers_name with
+             | Epers_Birth | Epers_Baptism | Epers_Death | Epers_Burial
+             | Epers_Cremation ->
+                 let name =
+                   match evt.epers_name with
+                   | Epers_Birth -> "birth"
+                   | Epers_Baptism -> "baptism"
+                   | Epers_Death -> "death"
+                   | Epers_Burial -> "burial"
+                   | Epers_Cremation -> "creamation"
+                   | _ -> ""
```

Comment:
this match should be a function `epers_name_to_string` defined elsewhere
(also notice the typo on cremation fixed in my gwu fix PR)

Reply:
ok